specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/LinkedService.json

```diff
@@ -369,14 +369,17 @@
         },
         "password": {
           "description": "Password to access the Dynamics instance.",
-          "$ref": "../datafactory.json#/definitions/AzureKeyVaultSecretReference"
+          "$ref": "../datafactory.json#/definitions/SecretBase"
+        },
+        "encryptedCredential": {
+          "type": "object",
+          "description": "The encrypted credential used for authentication. Credentials are encrypted using the integration runtime credential manager. Type: string (or Expression with resultType string)."
```

Comment:
Is there a structure to this object?

Reply:
Are you referring to the Secret Base or encryptedCredential? The former is defined in ../datafactory.json already (not a new object introduced by this PR), the latter is a base64 encoded string, which is a common property for all linked services.

Reply:
I am referring to `encryptedCredential` which is modeled as an `object` here, if it is a string you may have to model it as a string instead. 
You can take a look at [this](https://github.com/Azure/azure-rest-api-specs/blob/2fb9a0b3b902335ff0b0033711c234431931ec9d/specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/2016-06-01/registeredidentities.json#L141) if it helps

Reply:
If it can be a string or some other type, you can leave it as an object here.

Reply:
It is a String actually. I noticed other Strings are defined as an object as well. Besides, the encryptedCredential is a common property for all linked services, they are all defined as an object.

Reply:
Thanks for verifying this. You can keep it as an `object`